pkg/etcd/etcd.go

```diff
@@ -892,3 +891,57 @@ func backupDirWithRetention(dir string, maxBackupRetention int) (string, error)
 	}
 	return backupDir, nil
 }
+
+func GetServerURLFromETCD(ctx context.Context, cfg *config.Control) (string, int, error) {
```

Comment:
Can you change all the GetServer stuff to GetApiServer so it's clear what we're getting?